modules/lti/src/components/EditForm.tsx

```diff
@@ -49,10 +49,24 @@ function parseMetadataCollectionKey(s: string): MetadataCollectionKey {
 }
 
 function collectionToOptions(collection: EventMetadataCollection, translatable: boolean, t: i18next.TFunction): OptionType[] {
-    return collectionToPairs(collection)
+    const pairs = collectionToPairs(collection);
+    //Check if we get a JSON object back, and that this contains the order field
+    //Use the order field in this case to determine to order of the items in the dropdown
+    if(pairs[0][0].startsWith("{") === true) {
+        if(Object.prototype.hasOwnProperty.call(JSON.parse(pairs[0][0]), "order") === true) {
+            return pairs
+                .sort((a,b)  => JSON.parse(a[0]).order - JSON.parse(b[0]).order)
+                .map(([k, v]) => [parseMetadataCollectionKey(k).label, v])
+                .map(([k, v]) => [translatable ? t(k) : k, v])
+                .map(([k, v]) => ({ value: v, label: k}));
+        }
+    }
+    //Otherwise just return an alphabetically sorted list
+    return pairs
         .map(([k, v]) => [parseMetadataCollectionKey(k).label, v])
         .map(([k, v]) => [translatable ? t(k) : k, v])
-        .map(([k, v]) => ({ value: v, label: k }));
+        .map(([k, v]) => ({ value: v, label: k}))
+        .sort((a,b) => a.label.localeCompare(b.label));
```

Comment:
Is there a way to pass the current locale as second argument to `localeCompare()`? With german locale on Chrome at least the sort order is not correct, see screenshot below (Dänisch is before Deutsch but should be other way).

![image](https://user-images.githubusercontent.com/26736/121368115-1eaf7200-c93b-11eb-9bc0-f805ad964b1c.png)

Reply:
I'm by no means a grammtics expert, but don't you treat Umlaute the same way as normal letters when sorting in german? So "Dänisch" is effectively "Danisch" and therefore should be before "Deutsch"?

Reply:
It seems that the sorting is correct in german, but by coincidence, according to the documentation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare Dänisch should come before Deutch in german locale, but not in the swedish locale for instance. I'll update the PR to include the second parameter.